lib/ansible/module_utils/urls.py

```diff
@@ -1418,6 +1418,7 @@ def url_argument_spec():
         force_basic_auth=dict(type='bool', default=False),
         client_cert=dict(type='path'),
         client_key=dict(type='path'),
+        deprecated_aliases=dict(alias='thirsty', version='2.13'),
```

Comment:
I wonder if this makes more sense as a list or something since more than one alias may need deprecation and might need a different deprecation version. If it was just this arg spec it probably wouldn't matter, but since basic.py is handling this other people are going to want to utilize it and it seems like it should be flexible.